test/models/submission_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class SubmissionTest < ActiveSupport::TestCase
+  test "submission requires a user_id" do
+    sub_w_user = build(:text_submission)
+    sub_no_user = build(:text_submission, user: nil)
+    assert sub_w_user.valid?
+    assert_not sub_no_user.valid?
+  end
```

Comment:
when you get to RSpec, check out adding the [`shoulda-matchers` gem](https://github.com/thoughtbot/shoulda-matchers) as well. This provides methods like `it { is_expected.to validate_presence_of(:user_id) }` and many other helpful testing methods.